tests/vmi_configuration_test.go

```diff
@@ -3391,3 +3391,28 @@ func createBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume
 
 	return virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dataVolume, metav1.CreateOptions{})
 }
+
+func getProcessName(pod *k8sv1.Pod, pid string) (string, error) {
+	if pid == "" {
+		return "", fmt.Errorf("PID cannot be empty")
+	}
+
+	virtClient, err := kubecli.GetKubevirtClient()
```

Comment:
why did you make this change? using the singleton `kubevirt.Client()` seems faster and safer.

Reply:
because we use this in other parts of the code so it look better to me, can revert back to kubevirt.Client()

Reply:
I think that we should converge to using the singleton `kubevirt.Client()` whenever possible, as it is faster and safer.

Reply:
My bad, I was under the impression that is was the singleton.

Reply:
so fix in all places in the file?